### PR TITLE
First attempt at week 11 notes

### PR DESCRIPTION
**Pull request by Tiberius202**
Hey Harrison, here is a rough draft for my week 11 notes. There are some big areas I am unsure about including the induction over equality at the end and the core ideas of dependent type theory bullet points

Style things I have to change.
Using \frac
The images for the universe

### CODE REVIEW

File: week11.tex [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+Last week we covered partiality as well as other effects in Call-by-Push-Value (CBPV). In order to adapt logical relations to partiality, we needed a lemma called compactness. In short, compactness says that if a program terminates, it does so in some finite number of steps. 
+
+Compactness: if force(susp$(x.C)) \mapsto^* ret(yes / no)$ then there exists an $n$ such that $force(susp^{(n)}(x.C)) \mapsto^* ret(yes / no)$
+
+There exists other ways to deal with partiality including Denotational semantics however this only pushes the problem away from the operational semantics without providing us any tools to handle effects or effect-ful control flow. For this course, we prefer to stay lower level and focus on actual evaluation with operational semantics.
+
+An alternative method that is a little more complicated but gives a little more insight into the problem is Pitts (spelling?). He uses a logical relation that is indexed by an upper bound on the number of steps that a program takes to terminate.
+
+Logical relations on partiality rely on fixed point induction. This gives rise to methods of interpreting types as "complete denotational partial orders".
```
Comment (ice1000) [inline week11.tex]: ''

Comment (Tiberius202) [reply week11.tex]: Not sure what the change is. Is it untrue to say they are complete denotational partial orders? I have only heard of denotational from this one lecture.

**Comment by Tiberius202** [comment]
Would it be possible to get the original latex to replace the universes.png from a TA. I don't know if I have access to the original latex.

File: week11.tex (hunk 1 above)
Comment (HarrisonGrodin) [reply week11.tex]: I think Tesla is recommending using
```
this kind of ``quotes in LaTeX''
```
instead of the `"` character, which renders weirdly.

**Comment by HarrisonGrodin** [comment]
@Tiberius202 Done!

**Comment by Tiberius202** [comment]
I think it's ready to go. Although there are definitely gaps like on what the last equation means as it was never covered in class

**Comment by ice1000** [comment]
> I think it's ready to go. Although there are definitely gaps like on what the last equation means as it was never covered in class

I think that's just a computation rule, since J is the eliminator of Id, it should compute on the constructor. It's similar to case(inl(u), x.a, x.b) computes to a[u/x]

**Review by HarrisonGrodin — APPROVED** [review]
The formatting is a bit messy, but I was able to fix most of the things. I think it's probably best to merge in the meantime, and we can improve this later if desired.

**Pull request merged**